# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xmms/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Well, the usual defects.

  - Rene Rebe <rene@exactcode.de>

--- xmms-1.2.10/libxmms/xmmsctrl.h.orig	2004-09-29 01:05:05.193394951 +0200
+++ xmms-1.2.10/libxmms/xmmsctrl.h	2004-09-29 01:05:23.017554101 +0200
@@ -85,7 +85,7 @@
 
 	
 #ifdef __cplusplus
-};
+}
 #endif
 
 /* Deprecated APIs */
